locations/spiders/benugo_gb.py

```diff
@@ -0,0 +1,19 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class BenugoGBSpider(SitemapSpider, StructuredDataSpider):
+    name = "benugo_gb"
+    item_attributes = {"operator": "Benugo", "operator_wikidata": "Q20746208"}
+    sitemap_urls = ["https://benugo.com/site-sitemap.xml"]
+    drop_attributes = {"twitter", "facebook", "ref"}
```

Comment:
I suggest to keep `ref`s, they are important. Also doubt if twitter and Facebook needs to be dropped.

Reply:
the refs are just the website URL, not anything special

also the twitter and facebook are just links to the benugo company facebook, not a specific branch.

I don't know the best practices for ATP, should these be kept?

Reply:
Yes, please keep `ref`s, https://github.com/alltheplaces/alltheplaces/blob/master/DATA_FORMAT.md (identifier section) explains why they are important.